src/base/utils/misc.cpp

```diff
@@ -345,30 +370,34 @@ bool Utils::Misc::isPreviewable(const QString &extension)
 
 // Take a number of seconds and return an user-friendly
 // time duration like "1d 2h 10m".
-QString Utils::Misc::userFriendlyDuration(qlonglong seconds)
+QString Utils::Misc::userFriendlyDuration(std::chrono::seconds seconds)
 {
-    if ((seconds < 0) || (seconds >= MAX_ETA))
+    using std::chrono::duration_cast;
+    using namespace std::chrono_literals;
+
+    if (seconds >= MAX_ETA)
         return QString::fromUtf8(C_INFINITY);
 
-    if (seconds == 0)
+    if (seconds.count() == 0)
         return "0";
 
-    if (seconds < 60)
+    if (seconds < 1min)
```

Comment:
these literals are from C++14, in case you wanted to still be compatible with C++11